src/statisticCollector.js

```diff
@@ -147,7 +147,9 @@ StatisticCollector.prototype.specDone = function(jasmineSpec, specMeta) {
       matcher: jasmineExpectation.matcherName,
       stack: jasmineExpectation.stack,
       stepIndex: jasmineExpectation.stepIndex,
-      screenshot: jasmineExpectation.screenshot
+      screenshot: jasmineExpectation.screenshot,
+      shortMessage: jasmineExpectation.shortMessage,
```

Comment:
those are used only in screenshotReport so why not generate in a callback there and then displayed only there ?. Why do we need to expose to generic place ?